rosdep/python.yaml

```
@@ -4891,6 +4891,11 @@ python3-asyncssh:
   debian: [python3-asyncssh]
   fedora: [python3-asyncssh]
   ubuntu: [python3-asyncssh]
+python3-attrs:
+  debian: [python3-attr]
+  fedora: [python3-attrs]
+  opensuse: [python3-attrs]
```

The bot thinks that this is also available on RHEL, so:

```suggestion
  fedora: [python3-attrs]
  opensuse: [python3-attrs]
  rhel: [python3-attrs]
```